### DISCUSSION TITLE
[2022/06/07] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#208] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/207 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by NotFounds** [comment]
## HTML解体新書-仕様から紐解く本格入門
https://www.amazon.co.jp/HTML%E8%A7%A3%E4%BD%93%E6%96%B0%E6%9B%B8-%E4%BB%95%E6%A7%98%E3%81%8B%E3%82%89%E7%B4%90%E8%A7%A3%E3%81%8F%E6%9C%AC%E6%A0%BC%E5%85%A5%E9%96%80-%E5%A4%AA%E7%94%B0-%E8%89%AF%E5%85%B8/dp/4862465277

オフィスにあったのでパラパラめくってみた
基本的なタグや要素の話からアクセシビリティ・文字コードみたいな話まで広く触れられていて良い
「タイトルと本文の行間をあけたいときに `br` 重ねずに css でスタイルしなさい」みたいな話も出てきていて正しく実装するためにも良さそう
frontend 触る人はみんな読んでほしいと思った（特に2,3年以内の人とかは知らない単語や概念に出会える気がする）

```
目次

1章 HTMLの基本概念
1-1 HTMLとは
1-2 HTMLの仕様
1-3 HTML標準化の歴史
1-4 ウェブアクセシビリティの基礎
1-5 URLの概要
1-6 HTTP
1-7 技術情報との関わり方

2章 HTMLマークアップのルール
2-1 HTMLの要素とタグの基本
2-2 属性
2-3 URLの応用
2-4 要素の入れ子と内容モデル
2-5 HTMLで扱える文字
2-6 文字参照
2-7 コメント
2-8 HTMLの細かい構文ルール

3章 HTMLの主要な要素
3-1 「HTMLの主要な要素」の読み方
3-2 ルート要素と文書のメタデータ
3-3 セクション
3-4 グルーピングコンテンツ
3-5 テキストレベルセマンティックス
3-6 リンク関連要素
3-7 編集
3-8 エンベディッドコンテンツ
3-9 テーブル
3-10 フォーム1
3-11 フォーム2
3-12 インタラクティブ要素
3-13 スクリプティング

4章 主要な属性とWAI-ARIA
4-1 グローバル属性
4-2 WAI-ARIA
4-3 ARIA利用時の注意点
4-4 WAI-ARIAの実践
```
